packages/SketchMorph2-Core.package/M2Backend.class/instance/rectangleBrushSize.st

```diff
@@ -1,3 +1,3 @@
 accessing
 rectangleBrushSize
-	^ (0.5@1) * self brushSize
\ No newline at end of file
+	^ (0.5@1) * self brushSize
```

Comment:
Spaces could be added.

Reply:
Good point. I'll look into it.

Reply:
I added them in [5dc5732](https://github.com/hpi-swa-teaching/SketchMorph2/pull/73/commits/5dc573231db622f3500259b7b2351838614966a9).